account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -47,23 +57,146 @@ describe("MA v2 Tests", async () => {
     const txnHash1 = provider.waitForUserOperationTransaction(result);
     await expect(txnHash1).resolves.not.toThrowError();
 
-    const newAddressBalance = await client.getBalance({
-      address: target,
+    await expect(await getTargetBalance()).toEqual(
+      startingAddressBalance + sendAmount
+    );
```

Comment:
Nit: this works, but it's better practice to write this as
```ts
await expect(getTargetBalance()).resolves.toEqual(...));
```
because it makes a more helpful message if the test fails.